canonical-websites/www.ubuntu.com
# make pie chart scss into a pattern, rename testimonials to fit conven…

## Done

* created _v1_pattern_pie_chart.scss
* renamed _v1_patterns_testimonials to _v1_pattern_testimonials to fit pattern

# QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/cloud/openstack) to see that the pie charts still look correct
